awx/ui_next/src/screens/ManagementJob/ManagementJobList/LaunchManagementPrompt.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Button, TextInput, Tooltip } from '@patternfly/react-core';
+import { RocketIcon } from '@patternfly/react-icons';
+
+import AlertModal from '../../../components/AlertModal';
+
+const clamp = (val, min, max) => {
+  if (val < min) {
+    return min;
+  }
+  if (val > max) {
+    return max;
+  }
+  return val;
+};
+
+function LaunchManagementPrompt({
+  i18n,
+  isOpen,
+  isLoading,
+  onClick,
+  onClose,
+  onConfirm,
+  defaultDays,
+}) {
+  const [dataRetention, setDataRetention] = useState(defaultDays);
+  return (
+    <>
+      <Tooltip content={i18n._(t`Launch management job`)} position="top">
+        <Button
+          aria-label={i18n._(t`Launch management job`)}
+          variant="plain"
+          onClick={onClick}
+          isDisabled={isLoading}
+        >
+          <RocketIcon />
+        </Button>
+      </Tooltip>
+      <AlertModal
```

Comment:
I got to thinking about this some more and had a chat w/ @trahman73. We actually _don't_ want a prompt here (anymore) so I'll remove it.

Reply:
Curious how this will work without the modal.  This is where the user specifies the number of days right?

Reply:
That is correct.

Reply:
Update: We discussed this some more offline w/ @trahman73 and @wenottingham 

Plan is to go with the prompting flow.